ztp/source-crs/PtpOperatorConfigForEvent.yaml

```diff
@@ -9,7 +9,6 @@ spec:
   daemonNodeSelector:
     node-role.kubernetes.io/$mcp: ""
   ptpEventConfig:
-    # example "2.0", default to "1.0" if not set
     apiVersion: $event_api_version
```

Comment:
I think this should be removed in 4.19+

Reply:
You mean line `apiVersion: $event_api_version` should be removed, right?

Reply:
master will be 4.19 right @lack ?

Reply:
@aneeshkp What do you think?